gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/resources/IosResources.kt

```diff
@@ -127,7 +127,8 @@ private fun Framework.getFinalResourcesDir(): Provider<Directory> {
             .zip(
                 providers.environmentVariable("CONTENTS_FOLDER_PATH")
             ) { builtProductsDir, contentsFolderPath ->
-                File("$builtProductsDir/$contentsFolderPath/$IOS_COMPOSE_RESOURCES_ROOT_DIR").canonicalPath
+                val pathSuffix = if (target.isMacTarget()) "/Resources" else ""
+                File("$builtProductsDir/$contentsFolderPath$pathSuffix/$IOS_COMPOSE_RESOURCES_ROOT_DIR").canonicalPath
```

Comment:
I don't like the idea to hardcode a mac specific path here.
It is better to make new function for mac targets and support cocoapods as well then.

Reply:
Fixed the path by changing `CONTENTS_FOLDER_PATH` to `UNLOCALIZED_RESOURCES_FOLDER_PATH`. Now it is the same for iOS and macOS.

Reply:
Also added a test for cocoapods (copied from iOS with minor changes), so I think all should work fine.